h/migrations/versions/615358b6c428_move_all_groups_to_default_organization.py

```diff
@@ -0,0 +1,60 @@
+"""
+Move all groups to default organization.
+
+Revision ID: 615358b6c428
+Revises: f052da9df33b
+Create Date: 2018-03-30 16:08:56.063673
+"""
+
+from __future__ import unicode_literals
+
+import logging
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+
+revision = '615358b6c428'
+down_revision = '8a7f31c4525d'
+
+
+log = logging.getLogger(__name__)
+
+
+Base = declarative_base()
+Session = sessionmaker()
+
+
+class Group(Base):
+    __tablename__ = 'group'
+
+    id = sa.Column(sa.Integer, primary_key=True)
+    organization_id = sa.Column(sa.Integer, sa.ForeignKey('organization.id'))
+    organization = sa.orm.relationship('Organization')
+
+
+class Organization(Base):
+    __tablename__ = 'organization'
+    id = sa.Column(sa.Integer, primary_key=True)
+    pubid = sa.Column(sa.Text)
+
+
+def upgrade():
+    session = Session(bind=op.get_bind())
+
+    default_org = session.query(Organization).filter_by(pubid='__default__').one()
+
+    count = 0
+    for group in session.query(Group):
+        group.organization = default_org
```

Comment:
This will reset the associated organization for any groups which already have an organization associated with them, of which we have at least one in production. If we assign third-party groups to organizations before running this migration, then we'll need to avoid touching them as well.